### DISCUSSION TITLE
Cumulative Curl model solve_for_points implementation

### DISCUSSION THREAD

**Discussion by reos-aburns** [NatLabRockies/floris#1038] [Q&A]
Hi,

Currently the solve_for_points function (used by sample_velocity_deficit_profiles) is only available for the Gauss, Jensen, and Empirical Gauss models. I was wondering if there would be an implementation for the Cumulative Curl ('cc') velocity model in the near future?

Many thanks.

**Comment by misi9170** [comment]
Hi @reos-aburns ,

Thanks for bringing this up! This hasn't been a priority for us, but I will look into this a little bit and see if this is something we can enable for the next FLORIS release. I'll get back to you soon.

Misha

**Comment by misi9170** [comment]
I've opened an issue for this feature #1039

**Reply by reos-aburns** [reply]
Thank you!
